organization/resources.rst

```diff
@@ -0,0 +1,118 @@
+.. _doc_resources:
+
+Resources
+=========
+
+This is a list of resources that people regularly refer to when contributing to Godot.
+Most of these resources are part of the `Godot organization on GitHub <https://github.com/godotengine>`__,
+but some are hosted elsewhere.
+
+Resources are sorted by alphabetical order.
+
+Benchmarks viewer
+-----------------
+
+- **Link:** `benchmarks.godotengine.org <https://benchmarks.godotengine.org/>`__
+- `Source code <https://github.com/godotengine/godot-benchmarks/tree/main/web>`__
+  (maintained by `Calinou <https://github.com/Calinou>`__)
```

Comment:
I'm not sure I would mention maintainers by name here. We don't do this for other areas of the engine either, and it's not super relevant to the technical information presented here.

...Though I guess there's a point to be made for it since these tools don't have teams maintaining them, so this is the only place we _can_ mention the maintainers.
So idk, anybody else have an opinion?